### PR TITLE
Add Dashboards for kubeops/scanner

### PR DESCRIPTION
**Pull request by ArnobKumarSaha**
Signed-off-by: Arnob kumar saha <arnob@appscode.com>

### CODE REVIEW

File: scanner/app-level-cves.json [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "editorMode": "code",
+          "expr": "sum by(severity) (scanner_appscode_com_image_cve_count_total * on(image) group_left scanner_appscode_com_image_lineage{group=\"$group\",kind=\"$kind\",exported_namespace=\"$ns\",name=\"$name\"})",
```
Comment (pkbhowmick) [inline scanner/app-level-cves.json:114]: There should be a description field. After adding description, you can see an i-button in each panel right corner.

**Review by pkbhowmick — CHANGES_REQUESTED** [review]

File: scanner/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+There are four dashboards to monitor scanning information
+
+- `Appscode / Scanner / Cluster CVEs`: Shows vulnerability info in cluster level.
+- `Appscode / Scanner / Namespace CVEs`: Shows vulnerability info in namespace level.
+- `Appscode / Scanner / Image CVEs`: Shows vulnerability info for a particular image.
+- `Appscode / Scanner / App CVEs`: Shows vulnerability info for a particular workload. Ex: MongoDB, Deployment, Replicaset etc.
+
+
+Note: These dashboards are developed in **Grafana version 9.3.1**
```
Comment (pkbhowmick) [inline scanner/README.md:11]: I wonder, grafana versioned 9.x.x dashboard will perfectly work in 7.x.x or 8.x.x grafana version?

Comment (pkbhowmick) [reply scanner/README.md:11]: @ArnobKumarSaha can you please check these dashboard json is also working in grafana 7.x.x and 8.x.x ?

**Review by ArnobKumarSaha — COMMENTED** [review]
Now I have re-created the Scanner Dashboards using `Grafana v7.5.11`.  Also tested that on `v8.2.3` & `v9.0.1`. Its working nicely !

**Pull request merged**